README.md

```diff
@@ -0,0 +1,27 @@
+## Welcome to Serverless Workflow
+
+**Serverless Workflow** is an open-source, vendor-neutral, and community-driven project that redefines workflow creation with its high-level Domain Specific Language (DSL). Whether you are a developer, architect, or enthusiast, Serverless Workflow enables you to effortlessly design and execute workflows in a serverless environment. 
+
+### Why Serverless Workflow?
+
+Serverless Workflow is designed for ease of use and scalability, offering an intuitive DSL that makes creating complex workflows simple. Our system is event-driven, service-oriented, and FaaS-centric, integrating seamlessly with various platforms and protocols like HTTP, GRPC, and OpenAPI. With robust features for error handling, time management, and scalability, our workflows are both reliable and adaptable across diverse environments.
+
+### Our Ecosystem
+
+As part of the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/), Serverless Workflow is hosted as a Cloud Native Sandbox project. We offer a comprehensive suite of components and tools:
+
+- **[Specification](https://github.com/serverlessworkflow/specification)**: The core syntax and semantics of the Serverless Workflow DSL.
+- **[SDKs](https://github.com/serverlessworkflow/specification#sdks)**: Access our SDKs for [Go](https://github.com/serverlessworkflow/sdk-go), [Java](https://github.com/serverlessworkflow/sdk-java), [.NET](https://github.com/serverlessworkflow/sdk-net), [Python](https://github.com/serverlessworkflow/sdk-python), and [TypeScript](https://github.com/serverlessworkflow/sdk-typescript) to interact with workflows in your preferred programming language.
+- **[Synapse](https://github.com/serverlessworkflow/synapse)**: Our official, open source, vendor-neutral runtime implementation, facilitating seamless deployment and operation of workflows.
+
+### Join Our Community
+
+We thrive on community contributions and are always eager to welcome new members. Whether you're interested in developing new SDKs, enhancing existing tools, or sharing your insights, there's a place for you here.
+
+Connect with us on:
+- [GitHub Discussions](https://github.com/serverlessworkflow/specification/discussions)
+- [Slack](https://slack.cncf.io/)
+- [X/Twitter](https://twitter.com/ServerlessWF)
```

Comment:
```suggestion
- [X/Twitter](https://twitter.com/CNCFWorkflow)
```

Reply:
Do we still have access to `CNCFWorkflow` ?

Reply:
Oh yeah, I've recovered the username.